style.css

```diff
@@ -515,3 +515,423 @@ footer {
   margin-left: auto;
   margin-right: auto;
 }
+
+#nav-bar {
+  display: none;
+}
+
+.under-img {
+  display: none;
+}
+
+.page2-header {
+  display: none;
+}
+
+.page3-header {
+  display: none;
+}
+
+.desk-about-me-paragraph {
+  display: none;
+}
+
+.sele {
+  display: none;
+}
+
+/* desktop version */
+
+@media (min-width: 730px) {
+  #nav-bar {
+    display: flex;
```

Comment:
- [ ] Kindly note that the Desktop breakpoint in the CSS should be in the range of `768px` or wider